Resources/pre-recording-checklist.md

```diff
@@ -0,0 +1,27 @@
+#  Podcasting Pre-recording Checklist
+
+
+## Audio
+- Check input and output of each host via their computer's sound preferneces
+- Don’t tap on desk, or move around in chair, or scratch your face while talking
+- Turn off air conditioning
+- Try not to use a computer with a spinning hard drive
+
+## Other
+- Welcome to Cast - Introduce each host to the Cast recording platform or whichever recording solution you're using 
+- ICE: In Case of an Emergency, have a plan to get reconnected. Its best to get connected via a quick and reliable IM platform like iMessage
+- Define Intros - Define the introduction for each host (e.g., "Our first host is Garric Nahapetian. Garric is an iOS Enginner at Tinder as well as the founder of Learn Swift City.")  
+- Define Audience - Ensure each host knows who the target audience is for the episode
+- Set Limits: Ask each host what they definitely DO and definitely do NOT want to talk about
+
+## Last Call
+- Allow each host one last chance to use the restroom or get a drink of water
+- Ask each host to turn their electronics on airplane and or slient mode and the computer they are using to record on do not disturb
+- Ensure that any phone or cellular device including Apple Watches are either on airplace mode or away from microphones
+- Remind each host that Its just a conversation, no need to apologize
+- Remind each host to try to not use no curse words
```

Comment:
To not use* instead of “to not use no”

Reply:
Or simply "Remind each host to try not to curse" 🙂

Reply:
🤦🏽‍♂️

Reply:
All good, no worries @garricn :)